docs/dev.rst

```diff
@@ -107,9 +107,43 @@ Sharing directories with containers
 
 It may sometimes be convenient to mount container directories on the host, for instance: for editing and debugging. Tutor provides different solutions to this problem.
 
+.. _mount_option:
+
+Bind-mount volumes with ``--mount``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``run``, ``runserver``, ``init`` and ``start`` subcommands of ``tutor dev`` and ``tutor local`` support the ``-m/--mount`` option (see :option:`tutor dev start -m`) which can take two different forms. The first is explicit::
+
+    tutor dev start --mount=lms:/path/to/edx-platform:/openedx/edx-platform lms
+
+And the second is implicit::
```

Comment:
I like how you structured this whole section. Very informative and easy to follow.